components/KiboFacet.vue

```diff
@@ -257,5 +256,12 @@ export default defineComponent({
   --filter-count-margin: 0 0.5rem 0 auto;
 
   padding: 0.375rem;
+
+  @include for-mobile {
```

Comment:
default style should be mobile hence avoid using for-mobile

Reply:
Resolved